pass down $LDFLAGS to make sure libraries (in particular libpng) is picked up from the right location
author: Miguel Dias Costa (National University of Singapore)
updated to 1.3.6: micketeer@gmail.com
--- makeinclude.in.orig	2021-07-02 03:27:44.304515018 +0200
+++ makeinclude.in	2021-07-02 03:28:28.235425984 +0200
@@ -91,7 +91,7 @@
 CAIROFLAGS	= @CAIROFLAGS@
 DSOFLAGS	= -L. @DSOFLAGS@
 LDFLAGS		= $(OPTIM) @LDFLAGS@
-LDLIBS		= @LIBS@
+LDLIBS		= @LDFLAGS@ @LIBS@
 GLDLIBS		= @GLLIBS@ @LIBS@
 LINKFLTK	= @LINKFLTK@
 LINKFLTKGL	= @LINKFLTKGL@
